packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -128,7 +128,7 @@
 //! [`atomic`]: std::sync::atomic
 // This makes sure that `Emit` isn't regarded as dead-code even though it isn't exported on no-std.
 // This just simplifies maintenance, as otherwise we would be in cfg hell.
-#![cfg_attr(not(feature = "std"), allow(dead_code))]
+#![cfg_attr(not(feature = "std"), allow(dead_code, unreachable_pub))]
```

Comment:
This is because we only export this module on `feature = "std"`? Can you add this to the comment? Mayb also split the `#![cfg_attr]` and put a dedicated comment like
```suggestion
#![cfg_attr(not(feature = "std"), allow(dead_code))]
// This module is only exported with `std`. Without `std` the hooks are not available.
#![cfg_attr(not(feature = "std"), allow(unreachable_pub))]
```

Reply:
fixed in [3ca638d](https://github.com/hashintel/hash/pull/975/commits/3ca638d6f7029b2f91e046ac6badb66e28be2142)